datafusion/physical-expr/src/expressions/column.rs

```diff
@@ -136,6 +138,35 @@ pub fn col(name: &str, schema: &Schema) -> Result<Arc<dyn PhysicalExpr>> {
     Ok(Arc::new(Column::new_with_schema(name, schema)?))
 }
 
+// TODO: Move expressions out of physical-expr?
```

Comment:
to where would it go?

Reply:
leftover, it should be kept with `expressions/`